content/usage/concepts/matrix.md

```diff
@@ -76,8 +78,9 @@ pipeline:
       - go get
       - go build
       - go test
-+   environment:
+    environment:  
 +     - GO_VERSION=1.4
+-     - DATABASE=percona:5.5  
```

Comment:
I understand what this aims to illustrate, but it took me a while to figure it out. I think it might make the docs slightly more confusing, so perhaps we just stick to the bolded text we should hopefully be enough to warn the reader.

Reply:
that makes sense. I tried one more thing, but if that's still confusing I'll just take it out.